tests/syft/lib/torch/tensor_test.py

```diff
@@ -1,24 +1,58 @@
+"""In this test suite, we evaluate the operations done on a remote tensor 
+(we use the the pointer to a tensor).
+For more info on the remote tensor please see the documentation syft/lib/torch
+"""
+
+from syft.core.pointer.pointer import Pointer
+from syft.lib.torch.tensor_util import TORCH_STR_DTYPE
+
 import syft as sy
 import torch as th
 import pytest
+from itertools import product
 
+TYPES_TEST = [e for e in TORCH_STR_DTYPE.keys() if not e.startswith("complex")]
```

Comment:
Q: @tudorcebere @iamtrask do we want to test all types of tensors that we can send and do operations with them?

Reply:
Would be really cool!

Reply:
Absolutely

Reply:
Ideally we'd also test them in the context of backpropagation if we can.